### ISSUE TITLE
Add button to vdisk overview to checkup the DTL of the vdisk

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#1244]
It would be nice to be able to instantly checkup the DTL of a volume that has a degraded DTL as seen here:
![](http://pix.toile-libre.org/upload/original/1481018882.png)

The code is already in position to execute this per vdisk: https://github.com/openvstorage/framework/blob/5099afe52d67ae72d14286357b7706223c7bfd39/ovs/lib/vdisk.py#L856

**Comment by wimpers** [comment]
Suggested icon 
http://fontawesome.io/icon/random/ or
http://fontawesome.io/icon/refresh/

**Comment by wimpers** [comment]
@khenderick  I assume there is no API yet to call ?

**Comment by khenderick** [comment]
@wimpers, no, there is none. There are a lot of actions appearing, maybe we should clean them up or move them to another page?

**Comment by wimpers** [comment]
@khenderick if you save the vdisk (management actions), would that push the save through to the DTL checkup? I know this is a workaround.

**Comment by kvanhijf** [comment]
Currently the DTL checkup is NOT triggered when saving a vDisk via the
management actions of that vDisk

**Comment by wimpers** [comment]
Would that be a possible short term workarouind?

**Comment by kvanhijf** [comment]
Yes, but who would be responsible for triggering such workaround?  If our guys should execute this, we might as well provide them with a codesnippet to trigger the DTL checkup for a single vDisk instead of implementing yet another workaround

```
from ovs.lib.vdisk import VDiskController
VDiskController.dtl_checkup(vdisk_guid=<guid>)
```
